src/control/client/connect.go

```diff
@@ -118,8 +118,8 @@ type Connect interface {
 	//StorageBurnIn() (ClientCtrlrMap, ClientModuleMap)
 	ListFeatures() ClientFeatureMap
 	KillRank(uuid string, rank uint32) ResultMap
-	CreatePool(*pb.CreatePoolReq) ResultMap
-	DestroyPool(*pb.DestroyPoolReq) ResultMap
+	PoolCreate(logging.Logger, *PoolCreateReq) (*PoolCreateResp, error)
```

Comment:
The underlying implementation of this interface should supply its own logger. client.NewConnect() accepts a log.Logger and sets it on the returned *connList. The API methods themselves shouldn't require that a logger be supplied.